backend/Startup.cs

```diff
@@ -63,6 +63,9 @@ public void ConfigureServices(IServiceCollection services)
 
       services.AddTransient<IAuthService, AuthService>();
       services.AddTransient<IUserService, UserService>();
+      services.AddTransient<IWhaleService, WhaleService>();
+      services.AddTransient<ILocationService, LocationService>();
+      services.AddTransient<ISightingService, SightingService>();
```

Comment:
Very particular of me, but I like these to be in alphabetical order 😅 
```suggestion
      services.AddTransient<IAuthService, AuthService>();
      services.AddTransient<ILocationService, LocationService>();
      services.AddTransient<ISightingService, SightingService>();
      services.AddTransient<IUserService, UserService>();
      services.AddTransient<IWhaleService, WhaleService>();
```